packages/identity/src/identity.js

```diff
@@ -6,8 +6,8 @@ import { obfuscateUserId } from './userId';
 import { createUserSecretB64 } from './userSecret';
 
 type PermanentIdentityTarget = 'user';
-type SecretProvisionalIdentityTarget = 'email';
-type PublicProvisionalIdentityTarget = 'email' | 'hashed_email';
+type SecretProvisionalIdentityTarget = 'email' | 'phone_number';
+type PublicProvisionalIdentityTarget = 'email' | 'hashed_email' | 'phone_number';
```

Comment:
This makes me wonder why `'hashed_email'` and not `'hashed_phone_number'`?

Wouldn't it be clearer to keep only `'hashed_*'` fields in public provisional identities in the long run?

Reply:
to keep only one "target"/"name" for verification methods, private and public provisional identities. We never want a non-hashed phone_number in public objects (clear PII). The name would be 'salted_hashed_phone_number' to be really accurate.
In total (between Core, server and Identity) we manipulate 3 derivation of a unique phone number:
- salted hash for permanent: generated client side by Core
- salted hash for provisional: mostly used as is in the Core. It's an opaque ID (we don't care about what's in it)
- simple hash for verification: is (should) be temporary. Only exist server-side

Reply:
I agree that the naming is confusing, I'd like to have a descriptive field name to avoid errors, which already happened in these MR and elsewhere!

Reply:
C.f. la TEP pour référence, si vous voulez changer ca: https://tanker.atlassian.net/wiki/spaces/TKR/pages/2539421705/TEP+330+-+1+-+Without+Public+API+Break =)

Reply:
@maximerety what do you think?

Reply:
@theodelrieu I don't really know what I can help with... If you think of something, we can have a coffee and talk about it.

Reply:
Changed the TEP, going with `hashed_phone_number`